yarn-project/circuits.js/src/abis/abis.test.ts

```diff
@@ -39,7 +39,7 @@ describe('abis wasm bindings', () => {
 
   // TODO: This test fails on CI since build-system is not updating the latest circuits wasm
```

Comment:
@spalladino Do you know if build-system is now updating the circuits wasm? I had a green CI after re-enablement of this test but this does not mean it will remain so when circuits change.

Reply:
@jeanmon yeah, this should definitely be fixed now! This is from a moment where the CI was a lot flakier than it is now.

Reply:
@spalladino Great and thx for the reply. I will remove these comments then.